tests/test_interface.rb

```diff
@@ -958,6 +959,44 @@ def test_interface_ipv4_arp_timeout
     assert_raises(RuntimeError) { nonvlanint.ipv4_arp_timeout = 300 }
   end
 
+  def test_interface_fabric_forwarding_anycast_gateway
+    # Setup
+    config('no interface vlan11')
+    int = Interface.new('vlan11')
+    config('feature fabric forwarding')
+    foo = VxlanGlobal.new
+    foo.anycast_gateway_mac = '1223.3445.5668'
+
+    # 1. Testing default
+    int.fabric_forwarding_anycast_gateway = false
+    assert_equal(int.default_fabric_forwarding_anycast_gateway,
```

Comment:
Please use the more specific `refute` method in this case.

https://github.com/cisco/cisco-network-node-utils/blob/release_1.2.0/docs/README-develop-best-practices.md#mt2